### ISSUE TITLE
Wrong description when adding a person

### ISSUE THREAD

**Issue by Jarrett0203** [Jarrett0203/pe#4]
## Steps to Reproduce
* When adding the sample command from the UG.
```
add n/James e/james@example.com ig/itsjameshere tg/itsjameshere mod/CS2103T mod/CS2101 cca/NUS Hackers r/Year 2 b/01/01/2000
```

## Expected
* `New person added: James; Phone: ; Email: james@example.com; Birthday: January 1, 2000; Social media: Instagram: itsjameshere    Telegram: itsjameshere; Address: ; Remarks: [Year 2]; Modules: [CS2103T][CS2101]; Ccas: NUS Hackers` 

## Actual
* `New person added: James; Phone: Optional.empty; Email: Optional[james@example.com]; Birthday: Optional[January 1, 2000]; Social media: Optional[Instagram: itsjameshere    Telegram: itsjameshere    ]; Address: Optional.empty; Remarks: [Year 2]; Modules: [CS2103T][CS2101]; Ccas: NUS Hackers`
* Basically should not have Optional in the log and should just provide the values.

## System
OS: Windows 10 Version 22H2
JDK: Azul Zulu JDK 11.60.19 (11.0.17), 64-bit
Resolution: 1920 x 1080

**Comment by nus-pe-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
Thank you for the feedback.

Disagree. Rejecting because it is expected output as per our UG. It is documented that there will be Optional in the log as per the screenshots. Changing this to feature flaw as it is something we can improve on to make the app catered to less technically-inclined users.


![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2223S2/pe/master/files/525a3133-e6b0-4e60-91c2-8cbf4578957d.png)

# Items for the Tester to Verify

## :question: Issue response

Team chose [`response.Rejected`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>

## :question: Issue type

Team chose [`type.FeatureFlaw`]
Originally [`type.FunctionalityBug`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>

## :question: Issue severity

Team chose [`severity.VeryLow`]
Originally [`severity.Low`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
